multicluster/apis/multicluster/v1alpha1/clusterinfoimport_types.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2022 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterInfoImportStatus defines the observed state of ClusterInfoImport
+type ClusterInfoImportStatus struct {
+	Conditions []ResourceCondition `json:"conditions,omitempty"`
+}
+
+// +genclient
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// ClusterInfoImport is the Schema for the clusterinfoimports API
```

Comment:
This comment has no useful information in my mind.

Reply:
sure, some of them are automatically added by kubebuilder, I will remove it.